src/index.js

```diff
@@ -258,7 +262,10 @@ function getCurrentBatch<K, V>(loader: DataLoader<K, V, any>): Batch<K, V> {
   if (
     existingBatch !== null &&
     !existingBatch.hasDispatched &&
-    (maxBatchSize === 0 || existingBatch.keys.length < maxBatchSize)
+    (maxBatchSize === 0 ||
+      (existingBatch.keys.length < maxBatchSize &&
+        (!existingBatch.cacheHits ||
+          existingBatch.cacheHits.length < maxBatchSize)))
```

Comment:
I'm curious.

According to the README:

> **maxBatchSize**
> Limits the number of items that get passed in to the batchLoadFn.

Any reasons which the `cacheHits` would be using the same limit?
Since `batchLoadFn` isn't called for the cache hits.

And if the maxBatchSize was an "output batch size", then I assume we would want the limit to be something like:

    (keys.length + cacheHits.length) < maxBatchSize

Reply:
I found the same issue when hitting the limit with cache hits, my batchLoadFn is always called with 1 key only.

Reply:
Did you get some response about this?